third_party/terraform/tests/resource_app_engine_flexible_app_version_sweeper_test.go

```diff
@@ -0,0 +1,105 @@
+package google
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func init() {
+	resource.AddTestSweepers("AppEngineFlexibleAppVersion", &resource.Sweeper{
+		Name: "AppEngineFlexibleAppVersion",
+		F:    testSweepAppEngineFlexibleAppVersion,
+	})
+}
+
+// At the time of writing, the CI only passes us-central1 as the region
+func testSweepAppEngineFlexibleAppVersion(region string) error {
+	resourceName := "AppEngineFlexibleAppVersion"
+	log.Printf("[INFO][SWEEPER_LOG] Starting sweeper for %s", resourceName)
+
+	config, err := sharedConfigForRegion(region)
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error getting shared config for region: %s", err)
+		return err
+	}
+
+	err = config.LoadAndValidate(context.Background())
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error loading: %s", err)
+		return err
+	}
+
+	// Setup variables to replace in list template
+	d := &ResourceDataMock{
+		FieldsInSchema: map[string]interface{}{
+			"project":  config.Project,
+			"region":   region,
+			"location": region,
+			"zone":     "-",
+		},
+	}
+
+	listTemplate := strings.Split("https://appengine.googleapis.com/v1/apps/{{project}}/services", "?")[0]
+	listUrl, err := replaceVars(d, config, listTemplate)
+	if err != nil {
+		log.Printf("[INFO][SWEEPER_LOG] error preparing sweeper list url: %s", err)
+		return nil
+	}
```

Comment:
These lines are in the sweeper template because the we won't know what the URL will look like until generation. Fortunately we with a handwritten sweeper we can just replace these lines with:
```
	listUrl := "https://appengine.googleapis.com/v1/apps/" + config.Project + "/services"

```